src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.unit.spec.jsx

```diff
@@ -194,8 +194,8 @@ describe('ClaimsAndAppeals component', () => {
             }),
           ).to.be.false;
         });
-        it('should render a loading spinner but no section headline while loading data', () => {
```

Comment:
I saw these tests fail once, possibly for similar reasons as the `HealthCare` tests that started this whole adventure.